toolium/test/pageelements/test_page_element.py

```diff
@@ -369,3 +371,63 @@ def test_get_attribute(driver_wrapper):
     RegisterPageObject(driver_wrapper).username.get_attribute('attribute_name')
 
     mock_element.get_attribute.assert_called_once_with('attribute_name')
+
+
+def test_automatic_context_selection_no_webview_context_available(driver_wrapper):
+    driver_wrapper.is_mobile_test = mock.MagicMock(return_value=True)
+    driver_wrapper.config = mock.MagicMock()
+    driver_wrapper.config.getboolean_optional.return_value = True
```

Comment:
use
`driver_wrapper.config.set('Driver', 'automatic_context_selection', 'true')`
instead of 
`driver_wrapper.config.getboolean_optional.return_value = True`
to view more easily which variable has been mocked

Reply:
changed